### PR TITLE
Fixed bugs with installation & stellar noise

### PR DESCRIPTION
**Pull request by ideasrule**
Previously, it wasn't possible to disable stellar noise, as the program simply ignores that parameter.

### CODE REVIEW

File: wayne/run_visit.py [hunk 1]
```diff
@@ -301,7 +302,7 @@ def run():
     obs.setup_observation(x_ref, y_ref, spatial_scan, scan_speed)
     obs.setup_simulator(sample_rate, clip_values_det_limits, threads)
     obs.setup_trends(ssv_gen, x_shifts, x_jitter, y_shifts, y_jitter)
-    obs.setup_noise_sources(sky_background, cosmic_rate, add_read_noise)
+    obs.setup_noise_sources(sky_background, cosmic_rate, add_read_noise, add_stellar_noise)
```
[the remark this answers is not in the document]
Comment (ryanvarley) [reply wayne/run_visit.py]: Does it make sense to have this as an option anymore though? i.e. wont this just add extra stellar noise, and setting it to False will still have stellar noise from how electrons are chosen?

[the remark this answers is not in the document]
Comment (ideasrule) [reply wayne/run_visit.py]: It seems like the option eventually gets propagated to exposure_generator.py:629:

if add_stellar_noise:
            counts = np.random.poisson(counts)  # Poisson noise
        else:
            counts = np.round(counts)  # Still need ints for electron thrower

So if I'm understanding this correctly, there's still a quantization noise due to the finite number of electrons, but no Poisson noise if add_stellar_noise is False.

**Comment by ryanvarley** [comment]
@ideasrule I'm having issues installing this after all this time. Did the readme instructions work for you? If not can you explain what you did and we can update the readme at the same time.

**Comment by ideasrule** [comment]
@ryanvarley What errors are you getting?  It's been a long time since I've installed this, so I don't remember exactly what problems I encountered.  I think the only main problem was wayne not finding pyparallel, which I fixed in this pull request.

**Pull request merged**